vm/isolated/isolated.go

```diff
@@ -79,11 +86,13 @@ func (pool *Pool) Count() int {
 
 func (pool *Pool) Create(workdir string, index int) (vmimpl.Instance, error) {
 	targetAddr, targetPort, _ := splitTargetPort(pool.cfg.Targets[index])
+	USBAuth := fmt.Sprintf("%s%s%s", "/sys/bus/usb/devices/", pool.cfg.USBDevNum[index], "/authorized")
```

Comment:
If think we need a check here that `USBDevNum` is not empty.

Reply:
Also, I think it makes sense to pass USB devices number through the `Pool` struct and then form this string in `repair()`.

Reply:
you mean, remove `usbAuth` and declare it in `repair()`?

Reply:
Yes. Pass `index` the same way you pass `USBAuth` right now.